dashboard/application/app/controllers/support_ticket_controller.rb

```diff
@@ -30,8 +31,9 @@ def create
     end
 
     rts = RequestTrackerService.new ::Configuration.request_tracker_config
+    rts.selected_queue = queue
```

Comment:
I had initially thought that the queue selection would make sense as part of the `SupportTicket` object, but I moved it to the `RequestTrackerService` object when I saw that only the latter has the information from the client configuration file, which is necessary in order to validate the queue selection. This way, the queue selection can be validated at the time of assignment. If the queue selection is part of the `SupportTicket` object, then it cannot be validated until the `SupportTicket` object is passed to the `RequestTrackerService.create_ticket` method.

Do you think that it is ok to wait to validate the queue selection until this late in the execution flow?

Reply:
Ok. I was concerned it might be a problem if the `SupportTicket` object validates some of its instance variables at assignment and others are not validated until their use, but if you are ok with this then I will move the variable there.

Reply:
Done. 1653517dbfab8c5e7340b3c3064654fd88214c42